### PR TITLE
Added docker registry credentials

### PR DESCRIPTION
**Pull request by brunoOSI**
No story, fixes bubble downloading problems with proper docker credentials.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ node('intake-slave') {
     def failureColor = '#FF0000'
     SimpleDateFormat dateFormatGmt = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
     def buildDate = dateFormatGmt.format(new Date())
+    def docker_credentials_id = '6ba8d05c-ca13-4818-8329-15d41a089ec0'
```
Comment (TylerOSI) [inline Jenkinsfile]: is it okay to have this in our code? could someone use it to download db2data?

Comment (NagaPradeepGajula-osi) [reply Jenkinsfile]: Can we say that @brunoOSI

Comment (brunoOSI) [reply Jenkinsfile]: Yes. those are references to credentials [see the docs here](https://jenkins.io/doc/book/using/using-credentials/). 
They are already on [public repos](https://github.com/ca-cwds/dashboard/blob/master/Jenkinsfile#L3)

Comment (NagaPradeepGajula-osi) [reply Jenkinsfile]: cool I am waiting on this PR to get merged. so that I can get in my code merged in.

Comment (brunoOSI) [reply Jenkinsfile]: well, I need another approval 👍

**Pull request merged**